docs/python/tutorial-azure-functions.md

```diff
@@ -236,18 +236,40 @@ In these steps, you use the Functions extension to create a "Function App" on Az
     - In that resource group, create the storage account, App Service Plan, and App Service to host your code.
     - Deploy your code to the Function app.
 
-    You can also see progress in the **Azure: Functions** explorer:
+    The **Azure: Functions** explorer also shows progress:
 
     ![Deployment progress indicator in the Azure: Functions explorer](images/functions/deploy-progress.png)
 
-1. Once deployment is complete, the **Output** window shows the public endpoint on Azure:
+1. Once deployment is complete, the Azure Functions extension displays a message with buttons for three additional actions:
+
+    ![Message indicating successful deployment with additional actions](images/functions/deployment-popup.png)
+
+    For **Stream logs** and **Upload settings**, see the next sections. For **View output**, see step 5 that follows.
+
+1. After deployment, the **Output** window also shows the public endpoint on Azure:
 
     ```
     HTTP Trigger Urls:
       HttpExample: https://vscode-azure-functions.azurewebsites.net/api/HttpExample
     ```
 
-1. Use this endpoint to run the same tests you did locally, using URL parameters and/or requests with JSON data in the request body. You should see the same results from the public endpoint as you did locally.
+    Use this endpoint to run the same tests you did locally, using URL parameters and/or requests with JSON data in the request body. The results of the public endpoint should match those when you ran the function locally.
+
+### Stream logs
+
+> **Note**: support for log streaming is currently in development, as described on [Issue 589](https://github.com/microsoft/vscode-azurefunctions/issues/589) for the Azure Functions extension.
+
+The **Stream logs** button in the deployment message popup connects the log output on Azure to VS Code. You can also start and stop the log stream on the **Azure Functions** explorer by right-clicking the Functions project and selecting **Start streaming logs** or **Stop streaming logs**.
+
+When you start streaming logs, VS Code may prompt for consent, after which the log stream from Azure appears in a VS Code output window. (Again, this feature is in development, see [Issue 589](https://github.com/microsoft/vscode-azurefunctions/issues/589)).
```

Comment:
Copying Asavari's comment:

For Python, the streaming logs will not appear in the Code output window. Instead, they'll pop open the "Live Metrics Stream" view in the browser. 

While the VS Code experience is still in development, you can try this today using the following Core Tools command: 

```bash
func azure functionapp logstream APP_NAME --browser
```